modules/apps/27-interchain-accounts/controller/keeper/migrations.go

```diff
@@ -58,6 +58,9 @@ func (m Migrator) MigrateParams(ctx sdk.Context) error {
 		m.keeper.legacySubspace.GetParamSet(ctx, &params)
 
 		m.keeper.SetParams(ctx, params)
+
+		logger := m.keeper.Logger(ctx)
+		logger.Info("successfully migrated ica/controller submodule to self-manage params")
```

Comment:
since `logger` is only used once wdyt about `m.keeper.Logger(ctx).Info("...")` calls instead?